addons/wowl/static/src/services/rpc.ts

```diff
@@ -105,30 +99,28 @@ function jsonrpc(
       const { code, data: errorData, message, type: subType } = responseError;
       const { context: data_context, name: data_name } = errorData || {};
       const { exception_class } = data_context || {};
-      const name = exception_class || data_name;
-
-      const error: RPCServerError = {
-        type: "server",
-        code,
-        message,
-        data: errorData,
-        name,
-        subType,
-      };
-
-      bus.trigger("RPC_ERROR", error);
+      const exception_class_name = exception_class || data_name;
+
+      const error = new RPCError();
+      if (name && env.registries.errorDialogs.contains(exception_class_name)) {
```

Comment:
it seems to me that you don't really need the `alternativeComponent` mechanism. Can't the crash manager simply do the lookup?

Reply:
@aab-odoo Didn't use mention that having this alternativeComponent feature was useful in some cases ? Or maybe it was @kebeclibre ? Otherwise, yes, we could do the lookup somewhere else anyway, whatever the answer.

Reply:
Linked: https://github.com/odoo-dev/odoo/pull/509#discussion_r533409881